nvflare/private/fed/server/server_commands.py

```diff
@@ -423,6 +423,32 @@ def process(self, data: Shareable, fl_ctx: FLContext):
         return "Success"
 
 
+class ConfigureJobLogCommand(CommandProcessor):
+    """To implement the configure_job_log command."""
+
+    def get_command_name(self) -> str:
+        """To get the command name.
+
+        Returns: AdminCommandNames.CONFIGURE_JOB_LOG
+
+        """
+        return AdminCommandNames.CONFIGURE_JOB_LOG
+
+    def process(self, data: Shareable, fl_ctx: FLContext):
+        """Called to process the configure_job_log command.
+
+        Args:
+            data: process data
+            fl_ctx: FLContext
+
+        """
+        engine = fl_ctx.get_engine()
+        try:
+            dynamic_log_config(data, engine.get_workspace(), fl_ctx.get_job_id())
+        except Exception as e:
+            return secure_format_exception(e)
```

Comment:
when exception happen, this error message is returned, where in the code do we use this?
Do we print it to the users?

Reply:
`err = engine.configure_job_log(str(job_id), config)`
https://github.com/NVIDIA/NVFlare/pull/3127/files#diff-4cbbe70249ec6ca63b6c3172e6bc98ff066509e5c3bd3773e2790e6503d96849R366